examples/analytics-resource-timing.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
```

Comment:
Please move the test file to `test/manual` folder. Also to confirm the output `https://raw.githubusercontent.com/ampproject/amphtml/master/examples/img/ampicon.png?rt=analytics.script.k.1d` looks like a correct output?

Reply:
Done